kernel/realsense/d4xx.c

```diff
@@ -5414,6 +5421,7 @@ static int ds5_probe(struct i2c_client *c, const struct i2c_device_id *id)
 #endif
 	if (!state)
 		return -ENOMEM;
+	memset(state, 0, sizeof(*state));
```

Comment:
Why do we wish to zero this struct here?

Reply:
The memory block has been zero-initialized by devm_kzalloc(). I will remove this line.